package.json

```diff
@@ -12,7 +12,7 @@
     "dist"
   ],
   "scripts": {
-    "build": "node scripts/build.js",
+    "build": "RSPACK_CONFIG_VALIDATE=loose-silent node scripts/build.js",
```

Comment:
rspack doesn't support all webpack options and we do schema check for unsupported options, so we bypass schema check by settting RSPACK_CONFIG_VALIDATE=loose-silent